tests/test_CompareEclipseRestart.cpp

```diff
@@ -0,0 +1,173 @@
+#include "test_CompareEclipseRestart.hpp"
+
+int main(int argc, char** argv) {
+    char* gridFile1 = argv[1];
+    char* gridFile2 = argv[2];
+    char* unrstFile1 = argv[3];
+    char* unrstFile2 = argv[4];
+
+    if (argc != 5) {
+        printHelp();
+    }
+    else {
+        // Comparing grid sizes from .EGRID-files
+        {
+            ecl_grid_type* ecl_grid1 = ecl_grid_alloc(gridFile1);
```

Comment:
If I set `const ecl_grid_type* ecl_grid`, I get error in `ecl_grid_free(ecl_grid)` since this function changes `ecl_grid` I presume (?). I added const on the ints.